client/src/components/Home/PartnersSection/PartnersSection.css

```diff
@@ -54,33 +56,45 @@
       -ms-transform: translate(-50%, -50%);
       text-align: center;
 }
-
+.sponsors-logo-hover{
+display: none;
+}
 .logo-name {
       margin-left: auto;
       margin-right: auto;
       color: #fff;
-      font-weight: 200;
-      font-size: 1.9rem;
+      font-weight: 400;
+      font-size: 1.6rem;
       text-align: center;
-      width: 25vw;
+      width: 20vw;
       border: 1px solid #EE2A7B;
       border-radius: 20px;
       border-width: 1.6px;
       margin-bottom: 0;
 }
+.partners-link:hover{
 
+text-decoration: none;
+}
 .partners-col-1 {
+margin-left: auto;
+margin-right: auto;
       text-align: center;
       margin-bottom: 40px;
+column-count: auto;
 }

```

Comment:
Everything looks good to me 💯 just fix the indentation here. 
Suggestion : You can use `justify-content-center` class with `row` class for centering partners column instead of `margin-left:auto ; margin-right:auto;` 

![Screenshot (462)](https://user-images.githubusercontent.com/59157988/95020338-23bf7180-0688-11eb-91ac-1c6968634359.png)

Reply:
I dont think that's necessary here. Is there something I am missing?